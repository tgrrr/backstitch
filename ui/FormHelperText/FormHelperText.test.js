import React from 'react'
import renderer from 'react-test-renderer'
import { FormHelperText } from '.'

it('FormHelperText: default', () => {
    const component = renderer.create(
        <FormHelperText>
            <>
                FormHelperText
            </>
        </FormHelperText>
    )
  const tree = component.toJSON()
  expect(tree).toMatchSnapshot()
})
