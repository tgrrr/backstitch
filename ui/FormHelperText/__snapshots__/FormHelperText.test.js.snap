// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`FormHelperText: default 1`] = `
<div
  className="sc-AxjAm StDqN"
>
  <div
    className="Formhelpertext "
    data-testid="Formhelpertext"
  >
    <p
      className="MuiFormHelperText-root"
    >
      FormHelperText
    </p>
  </div>
</div>
`;
